Classes/Controller/BanController.php

```diff
@@ -1,6 +1,30 @@
 <?php
 namespace Aoe\Varnish\Controller;
 
+/***************************************************************
+ *  Copyright notice
+ *
+ *  (c) 2017 AOE GmbH <dev@aoe.com>
```

Comment:
Please use 2018 when adding new Copyrights. Applies to all files with new copyright.